### ISSUE TITLE
Check if executing in browser environment

### ISSUE THREAD

**Issue by michaelBenin** [nitin42/react-imgpro#2]
Server rendering yields:

`ReferenceError: window is not defined`

**Comment by michaelBenin** [comment]
`exenv` is a pretty good package for this.

**Comment by nitin42** [comment]
Yeah! I'll look into this later this week. Thanks for the suggestion 😄

**Comment by nitin42** [comment]
released `react-imgpro@1.1.0` 🔥 ! Should fix this... Let me know!

**Comment by nitin42** [comment]
Released `react-imgpro@1.2.0` ! Should fix this.. let me know!

**Comment by chris-mo** [comment]
I am still getting this issue even with 1.2.0

**Comment by nitin42** [comment]
Ohk! I got it. Need a runtime checker for non browser code. I'll post an update soon 😉

**Comment by nitin42** [comment]
Done ✅

**Comment by chris-mo** [comment]
Thank you for the quick update!

**Comment by nitin42** [comment]
Is it working now?
